### ISSUE TITLE
How to get Canteens of purified water is not clear.

### ISSUE THREAD

**Issue by The-Arcanian** [Stereowalker/Survive#56]
As I expressed here: https://github.com/Stereowalker/Survive/issues/8. I tried multiple methods to try and get a canteen of purified water because nothing told me how, up to the point that I even asked on an old issue.

**Comment by Oriansenshi** [comment]
you put a filled canteen in a crafting grid with a purification filter. It is very simple, but also not clearly stated at all. I feel like there needs to be an in game survival guide or something.

**Comment by The-Arcanian** [comment]
Agreed, it is simple. But due to the lack of clarity, I tried a variety of other things before I thought of it. Honestly, the answer may be as simple as a tooltip on the Charcoal Filter like "Purifies Canteens".

**Comment by iobiarg** [comment]
thanks, i've been looking for this information for a week or more in-game.... there should be a guide or sth, I didnt know what was the filter for XD

**Comment by Oriansenshi** [comment]
Also, I just found out that if you craft the filter with a bucket of water you get a bucket of purified water. I think it works with a bowl too, but I havent tested it yet. You unfortunately cannot make an infinite purified water source, which I think would be great to have in a base. I still have to test to see if you can fill a cauldron or something with it.

**Comment by iobiarg** [comment]
It would be awesome to have some way of getting either big amounts of water purified or a source...
It could be from a safe source of water like in high mountains, or cenotes. 
It could be with some sort of furnace with a pot over it (like in Farmers Delight) or sth like that, i don't want a machine but sth more medieval. Even a tank were to throw the filter maybe?
